AutoLayoutHelper.xcodeproj/project.pbxproj

```diff
@@ -346,6 +347,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				INFOPLIST_FILE = AutoLayoutHelper/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.madhikarma.$(PRODUCT_NAME:rfc1034identifier)";
```

Comment:
I looked through the original diff and it was previously set as com.madhikarma so nothing to do with Xcode 7 just something I didn't realise when we brought it back to ustwo.

Next Pull Request will add a target to create a dynamic framework so its easier to integrate